home/import_content_pages.py

```diff
@@ -281,8 +281,11 @@ def create_content_page_index_from_row(self, row: "ContentRow") -> None:
                 self.home_page(locale).add_child(instance=index)
             index.save_revision().publish()
         except ValidationError as err:
-            # FIXME: Find a better way to represent this.
-            raise ImportException(f"Validation error: {err}")
+            for key in err.error_dict:
+                field_name = key
+            raise ImportException(
+                f"Validation error: {field_name} - {'; '.join(err.messages)}"
```

Comment:
If we're getting more than one of these, perhaps add them each as a separate message like we do with the wagtail validation errors?